libs/h3/src/main/java/org/elasticsearch/h3/LatLng.java

```diff
@@ -74,6 +79,50 @@ public double getLonDeg() {
         );
     }
 
+    /**
+     * Computes the point on the sphere with a specified azimuth and distance from
+     * this point.
+     *
+     * @param az       The desired azimuth.
+     * @param distance The desired distance.
+     * @return The LatLng point.
+     */
+    LatLng geoAzDistanceRads(double az, double distance) {
```

Comment:
This method seems to only be used in tests to compare with the results from `Vec3d.geoAzDistanceRads`. Is it intended to be used in anything else but tests?

Reply:
I think it is still valuable to have it around, maybe there are situations (e,g .you have lat/lng but not the x/y/z representarion) where it might be useful.